.travis.yml

```diff
@@ -20,7 +20,7 @@ env:
   - DJANGO="django>=1.11,<1.12"
   - DJANGO="django>=2.0,<2.1"
 
-matrix:
+jobs:
```

Comment:
I checked this is only appearing as `info` not as `warning` so I think we can reverse it for now. 
(The thing is that these things keep changing, so it's not worth our time to keep the `info` part changing, only when it appears as a `warning` that we need to do something about it! :smile: )

Reply:
I have corrected and reversed the info parts